CONTRIBUTING.md

```diff
@@ -49,3 +49,10 @@ branches.
 
 Please see our [git style guide](https://github.com/alphagov/styleguides/blob/master/git.md)
 which describes how we prefer git history and commit messages to read.
+
+### Review apps
+
+When a pull request is opened Heroku will create a [review app](https://devcenter.heroku.com/articles/github-integration-review-apps#viewing-review-apps)
+that will allow you and reviewers to preview how your changes will appear for users.
```

Comment:
This is only true for PRs raised internally:

> For security and billing reasons, Heroku does not automatically create review apps for pull requests to public repos that are sent from forks.

For external contributors (which I guess is the target audience for this documentation) the review apps need to be manually created by us.

Reply:
Oh, I did not know that. Well I guess it makes sense.

Do you think then it is worth having documentation for external contributors?

Or we could just put the info about username and password in the internal docs along with info on how to create a review app for an external PR.

Reply:
Good q. Maybe we could say something like 'Review apps can be created by the Prototype Kit team on request'?

Reply:
Added a line to that effect.